src/components/sidebar/Sidebar.jsx

```diff
@@ -85,6 +84,17 @@ const Sidebar = ({ menuOpened, setMenuOpened }) => (
         <li className="nav-item">
           <ThemeToggler />
         </li>
+        <li className="nav-item">
+          <NavLink
+            to="/detail-page"
+            onClick={() => {
+              setMenuOpened();
+            }}
+          /* eslint-disable-next-line */
```

Comment:
- [ ] I know you had it for a reason but I removed it and now I think it's not needed, let's kindly remove it

Reply:
Thanks for the suggestion, it's fixed!